packages/api/amplify_api/example/integration_test/graphql/api_key_test.dart

```diff
@@ -10,14 +10,18 @@ import 'package:integration_test/integration_test.dart';
 
 import '../util.dart';
 
-void main({bool useExistingTestUser = false}) {
+void main({bool useExistingTestUser = false, bool useGen2 = false}) {
```

Comment:
Q: It looks like the PR updates all tests to pass `true` for `useGen2`. What is the benefit of including a flag to run the tests with the old CI stack?

I could see some benefit is being able to use the old config format, but not sure how much benefit we are getting from running the tests on the old stack.

Reply:
We discussed offline. IIRC the summary was that we are not running the tests on the old stack. We are generating the old config from the new stack. Long term we should look to increase the unit tests for the AmplifyConfig -> AmplifyOutputs mapping so that we do not have to rely on running integ tests for this.